controllers/apply/awards-for-all/fields/main-contact-phone.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+const PhoneField = require('../../lib/field-types/phone');
+
+module.exports = function(locale) {
```

Comment:
This looks a little silly, but felt worth being consistent even if it means the definition is small.